### DISCUSSION TITLE
Compare speed very slow when compare local foler with Shared folder in LAN

### DISCUSSION THREAD

**Discussion by swpatrick** [WinMerge/winmerge#819] [Q&A]
So I have 2 PC, the file/folder transfer speed(like copy and paste speed in file explorer) from LAN shared folder to Hard drive is about the same, but when I use WinMerge to compare two folders (one in hard drive, other in LAN shared folder) there is a huge speed difference, the fast one is like 20s and the slow one is about 400s, does anyone know what cause this issue? Thanks

**Comment by TheJPomp** [comment]
Are they folders containing many subfolders that also contain many subfolders that will contain files? I ask because my network comparisons are super slow because of this. Gigs of small files in nested folders will take time.

What I would like is the ability to poll a folder locally, then export that result to a file that can be brought to the 2nd computer and compare against there, without needing to compare across the network.

**Reply by swpatrick** [reply]
yes, the folder structure is like what you said. I don't unserstand the second part of you answer.

**Comment by mlipok** [comment]
Turn off yours AV software, the any LAN action will speed up.

**Reply by swpatrick** [reply]
well, both machines running the same AV software, if AV is the reason, why does it only slow down compare speed on 1 of the machine, anyway I will try to turn AV off to see does it make difference.
